### ISSUE TITLE
Monkeypatch fail on pypy 7.1

### ISSUE THREAD

**Issue by carsonip** [gevent/gevent#1380]
* gevent version: 1.3.7
* Python version: PyPy 7.1.0
* Operating System: Ubuntu 16.04

### Description:

Just started playing with PyPy. Realize gevent monkey-patching doesn't work. Is this expected? Thanks.

```
Python 2.7.13 (990cef41fe11, Mar 21 2019, 12:14:52)
[PyPy 7.1.0 with GCC 6.2.0 20160901] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>>> import gevent.monkey
>>>> gevent.monkey.patch_all()
Traceback (most recent call last):
 File "<stdin>", line 1, in <module>
 File "venvpypy/site-packages/gevent/monkey.py", line 976, in patch_all
   patch_thread(Event=Event, _warnings=_warnings)
 File "venvpypy/site-packages/gevent/monkey.py", line 178, in ignores
   return func(*args, **kwargs)
 File "venvpypy/site-packages/gevent/monkey.py", line 588, in patch_thread
   _patch_existing_locks(threading_mod)
 File "venvpypy/site-packages/gevent/monkey.py", line 492, in _patch_existing_locks
   if o._RLock__owner is not None:
AttributeError: 'thread.RLock' object has no attribute '_RLock__owner'
```
```
>>>> dir(threading.RLock())
['__class__', '__delattr__', '__doc__', '__enter__', '__exit__', '__format__', '__getattribute__', '__hash__', '__init__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__str__', '__subclasshook__', '__weakref__', '_acquire_restore', '_is_owned', '_note', '_release_save', 'acquire', 'release']
```

**Comment by carsonip** [comment]
I did a bit of digging around, appears to me that it is related to: https://bitbucket.org/pypy/pypy/commits/4874b6c33f9528704320ff870cc09e5c1e62f040

**Comment by carsonip** [comment]
Duplicate of #1357, fixed in d0e0465891e30abb3e64c25324268251ad5c994c. Sorry for this duplicate issue.
